testing/unittests/jlibtests.cpp

```diff
@@ -43,11 +45,9 @@ class JlibSemTest : public CppUnit::TestFixture
 
     void testTimedAvailable(Semaphore & sem)
     {
-        unsigned now = msTick();
-        sem.wait(100);
-        unsigned taken = msTick() - now;
         //Shouldn't cause a reschedule, definitely shouldn't wait for 100s
-        ASSERT(taken < 5);
+        if(!sem.wait(100))
+            ASSERT(false)
```

Comment:
missing ';' (seems to compile anyway, but for consistency)

Reply:
Changed